ir/pattern/constraint.rs

```diff
@@ -2696,3 +2710,62 @@ impl<ID: IrID> fmt::Display for LinksDeduplication<ID> {
         write!(f, "LinksDeduplication({}, {}))", self.links1, self.links2)
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct OptimisedToUnsatisfiable {
+    conjunction: Conjunction,
+}
```

Comment:
Can this just hold the scope id? It doesn't look like the conjunction is used anywhere.

Reply:
^ We're deprecating + this might be useful for debugging?